test/BaselineOutput/Common/FeatureSelection/countFeatureSelect.tsv

```diff
@@ -0,0 +1,16 @@
+#@ TextLoader{
+#@   sep=tab
+#@   col=ScalarFloat:R4:0
+#@   col=VectorFloat:R4:1-4
+#@   col=VectorDouble:R8:5-9
+#@   col=FeatureSelect:R4:10-13
+#@   col=FeatureSelectDouble:R8:14-18
+#@   col=ScalFeatureSelectMissing100:R4:19
+#@   col=VecFeatureSelectMissing100:R8:20-24
+#@   col=ScalFeatureSelectMissing690:R4:25
+#@   col=VecFeatureSelectMissing690:R8:26-29
```

Comment:
wonder why the order here is different from the order specified in the CountFeatureSelectionWorkout  test ?

Reply:
Some columns are copied if no slots are dropped. All the copy are grouped and all the drop operations are grouped, this is why you see a difference in the order. However, I don't think that the order matters in this case. What matters is that the correct columns are present in the output. 

---
In reply to: [235552541](https://github.com/dotnet/machinelearning/pull/1683#discussion_r235552541) [](ancestors = 235552541)